util/helm.go

```diff
@@ -107,3 +107,12 @@ func CopyDir(src, dst string) error {
 	filepath.Walk(src, walker)
 	return failure
 }
+
+//CopyFile copies file from src to dst
+func CopyFile(src string, dst string) error {
+	data, err := ioutil.ReadFile(src)
```

Comment:
Makes sense when your file is large. This utilitu function I wrote just for the test. Which is a couple of lines. I mean we are not talking about MBs of files. But still let me know if you want me to change